lib/internal/abort_controller.js

```diff
@@ -96,6 +99,12 @@ const kComposite = Symbol('kComposite');
 const kSourceSignals = Symbol('kSourceSignals');
 const kDependantSignals = Symbol('kDependantSignals');
 
+const converters = {};
+converters.any = (V) => {
+  return V;
+};
```

Comment:
I don't understand why we need this, could you explain? In any case, it should be declared in one instructions to avoid prototype pollution.
```suggestion
const converters = {
  any = (V) => V,
};
```

Reply:
> I don't understand why we need this, could you explain?

Sure, the `createSequenceConverter` accepts an converter so that we can transform the iterable to an array. The passed-in converter is used to do conversion for each element in that sequence (iterable in this case). I used the `any` converter compliant with [the spec](https://webidl.spec.whatwg.org/#es-any). Undici does the same for this [here](https://github.com/nodejs/undici/blob/3f1b57d28fc084b1ac712a7ca844b1b3fef77be8/lib/web/fetch/webidl.js#L516).

> In any case, it should be declared in one instructions to avoid prototype pollution.

I agree! Maybe it's also better to put the spec link as a comment?

After thinking about it, it probably would also be better to lift this out to a common module like `internal/webidl.js`

Reply:
I was wondering why the `webidl.js` lives in `crypto` with many spec compliant converters, then I traced back to [this comment](https://github.com/nodejs/node/pull/46067#issuecomment-1370017660), which all makes sense.

I think it is now a good time to start lifting some common converters out so that other modules can benefit from it? I can do some follow-up PRs for this.

I checked deno implementation, [they used `createInterfaceConverter` rather than `any`](https://github.com/denoland/deno/blob/3a1a1cc030fb7fc90d51ee27162466d6ac924926/ext/web/03_abort_signal.js#L306) to validate `AbortSignal` instance in the incoming sequence. However, we have checker that has done that, which I way I simply pass an `any` converter to convert the sequence to array
https://github.com/nodejs/node/blob/298dea0c635e776cf00fc9014e6e4ce2f760e039/lib/internal/validators.js#L364

Reply:
The WebIDL in SubtleCrypto is a cut-down version, often without the necessary options required for other specs or even only implementing a particular WebIDL option that is used by SubtleCrypto.

If there was an effort to implement performance-optimized and compliant WebIDL for all of Web Platform API in Node.js to make use of I would NOT recommend starting with that as a baseline for such effort.

Reply:
Aggregated this to `internal/webidl.js`. There is a base convereters avoiding prototype pollution